seat.c

```diff
@@ -6,11 +6,14 @@
  * See the LICENSE file accompanying this file.
  */
 
+#define _XOPEN_SOURCE 500
```

Comment:
If this is for `strdup`, then `#define _POSIX_C_SOURCE 200809L` should be enough.